change/@fluentui-react-charting-e4dc8ea7-f277-4bf1-a526-a85995f9f5e0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30346&commitId=6b0ba16840e42d48d9265bb81a139c2c1a83219a&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-HorizontalBarChart</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HorizontalBarChart.Variant - RTL.default.chromium.png|73895| Changed|
|react-charting-HorizontalBarChart.Variant - Dark Mode.default.chromium.png|31249| Changed|
|react-charting-HorizontalBarChart.Variant.default.chromium.png|75557| Changed|
|react-charting-HorizontalBarChart.With_Benchmark - Dark Mode.default.chromium.png|31968| Changed|
|react-charting-HorizontalBarChart.With_Benchmark - RTL.default.chromium.png|86074| Changed|
|react-charting-HorizontalBarChart.With_Benchmark.default.chromium.png|90375| Changed|
|react-charting-HorizontalBarChart.Basic - RTL.default.chromium.png|151862| Changed|
|react-charting-HorizontalBarChart.Basic - Dark Mode.default.chromium.png|48424| Changed|
|react-charting-HorizontalBarChart.Basic.default.chromium.png|158297| Changed|
|react-charting-HorizontalBarChart.Basic - Dark Mode.hover.chromium.png|0| Removed|
|react-charting-HorizontalBarChart.Basic.hover.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>